Add Nebius Embedder documentation

Nancy-Chauhan opened the pull request:
## Description
Add Nebius Embedder documentation

**Note:** Your PR title must follow conventional commit format (e.g., `docs: add authentication guide`, `fix: correct broken links`, `style: update formatting`). See [CONTRIBUTING.md](../CONTRIBUTING.md) for details.

## Type of Change
- [ ] Bug fix (errors, broken links, outdated info)
- [x] New content
- [ ] Content improvement
- [ ] Other: ____

## Related Issues/PRs (if applicable)
<!-- Link any related items -->
- Closes #____
- Related SDK PR: agno-agi/agno#____

## Checklist
- [x] Content is accurate and up-to-date
- [x] All links tested and working
- [x] Code examples verified (if applicable)
- [x] Spelling and grammar checked
- [ ] Screenshots updated (if applicable)

Nancy-Chauhan committed:
Add Nebius Embedder documentation

dirkbrnd commented on _snippets/embedder-nebius-reference.mdx:
Please don't use snippets, they are broken in mintlify. Rather just copy the code into the reference

dirkbrnd commented on concepts/knowledge/embedder/nebius.mdx:
````diff
@@ -0,0 +1,51 @@
+)
+```
+
+## Params
````
Lets rather just link to the reference? If the other pages are like this, then they should also be updated

Nancy-Chauhan committed:
Remove snippets and link to reference page for Nebius embedder

- Replace snippet include with actual parameters in reference page
- Replace parameters table with reference link in concepts page
- Delete unused snippet file
